xml/System.Numerics.Tensors/Tensor.xml

```diff
@@ -3975,9 +3985,9 @@
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="lengths">To be added.</param>
-        <param name="pinned">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="lengths">A <see cref="T:System.ReadOnlySpan`1" /> indicating the lengths of each dimension.</param>
+        <param name="pinned">A <see cref="T:System.Boolean" /> whether the underlying data should be pinned or not.</param>
```

Comment:
```suggestion
        <param name="pinned">
          <see langword="true" /> to pin the underlying data; otherwise <see langword="false" />.</param>
```